src/firefly/java/edu/caltech/ipac/util/FileUtil.java

```diff
@@ -71,7 +71,7 @@ public class FileUtil
     public static final long K            = 1024;
 
     private static String[] CHARSETS_TO_BE_TESTED = {"ISO-8859-1","windows-1252", "windows-1253","UTF-8", "UTF-16",};
-    public static final int BUFFER_SIZE = (int) (32 * FileUtil.K);
+    public static final int BUFFER_SIZE = (int) (16 * FileUtil.MEG);
```

Comment:
The buffer size you're suggesting is MUCH larger than the 8 KB default which is typically sufficient for many use cases. From the testing I did a while back, increasing the size beyond a certain point showed no performance gain. 
Since this is used throughout, I'd recommend testing to confirm if the larger buffer actually improves performance before committing to it.
Or, isolate it to the piece of code where you see this large buffer size improved performance.

Reply:
Seems on modern machines 8KB is pathetic. I don't understand why it can take advantage of a bigger buffer.

However, I can change it back since you test show no improvement.